system/indy-node-tests/test_vc.py

```diff
@@ -52,6 +52,25 @@
     await ensure_pool_is_functional(pool_handler, wallet_handler, trustee_did, timeout=360)
 
 
+@pytest.mark.asyncio
+async def test_vc_by_demotion_last(
+        pool_handler, wallet_handler, get_default_trustee, nodes_num, check_no_failures_fixture
+):
+    _alias = 'Node7'
+    _did = 'BM8dTooz5uykCbYSAAFwKNkYfT4koomBHsSWHTDtkjhW'
+    trustee_did, _ = get_default_trustee
+    await ensure_pool_is_functional(pool_handler, wallet_handler, trustee_did)
+    primary_first, _, _ = await get_primary(pool_handler, wallet_handler, trustee_did)
```

Comment:
I think adding line breaks between logical sections of test would improve readability

Reply:
Done.